_pages/docs/next-ls/editors.md

```diff
@@ -289,20 +297,7 @@ Zed will install and manage Next LS for you.
 }
 ```
 
-If you want to run a local version of `next-ls` you can do the following:
-
-```json
-"elixir": {
-  "lsp": {
-    "local": {
-      "path": "<PATH TO BINARY>",
-      "arguments": ["--stdio"]
-    }
-  }
-},
-```
-
-Where `PATH TO BINARY` is a location to a local build, eg. `/Users/user/next-ls/burrito_out/next_ls_darwin_arm64`.
+And, if you want to run a local version of `next-ls` make sure your local build is in your path named `next-ls`.
```

Comment:
the binary name should be `nextls`, not `next-ls`, I believe thats how its set up in zed.

Reply:
Yep, you're [correct](https://github.com/zed-industries/zed/blob/main/extensions/elixir/src/language_servers/next_ls.rs#L25). Fixed in [force push](https://github.com/elixir-tools/elixir-tools.dev/compare/41a013d8f2c139353f54c50b2d5d387ebd35d42e..46dfa91935700d9b854611909ff8d6e2f472e704)